src/Repository/CardRepository.php

```diff
@@ -23,4 +24,14 @@ public function findOneByName(string $name): ?Card
     {
         return $this->entityManager->getRepository(Card::class)->findOneBy(['name' => $name]);
     }
+
+    public function findOneById(int $id): Card
```

Comment:
:) body of this method is almost correct but name of it not correct. Names starts with `find...` suggest that we search something and if we do then we could find or not so method should return `Optional<Something>` so nullable value.

If you use method with name `get...` its mean that you want to take something from db that we assume exist, and in that case we expect that method return `Something` or Exception if was not found.